### DISCUSSION TITLE
Table 13, Panel 3

### DISCUSSION THREAD

**Discussion by joshichaitanya3** [cssi-pi-community/2022-meeting#90] [Panel 3 roundtable (Tuesday morning)]
This is the start of the discussion for Panel 3!

**Comment by inducer** [comment]
By way of introduction, I'm Andreas, also @inducer.

**Comment by amilworks** [comment]
My name is Amil and our platform is [BisQue](https://github.com/UCSB-VRL/bisqueUCSB)

**Comment by joshichaitanya3** [comment]
Hi! I am Chaitanya Joshi from Tufts University, working on the open-source language [Morpho](https://github.com/Morpho-lang/morpho), aimed at solving shape optimization problems.

**Comment by inducer** [comment]
A complicating factor is that project needs are not constant over the lifetime:
- rapid building at the start, often marked by lack of process
- more process needed as projects evolve, documentation, CI, testing

**Reply by inducer** [reply]
Can be hard to recover from early mistakes.

**Comment by inducer** [comment]
Attrition of knowledge can be an issue as sustaining group members leave and join.

**Comment by inducer** [comment]
Developer skill development represents a trade-off between mechanical skills (using Git, CI, testing, documentation) and "perspective"/"common sense".
